### PR TITLE
Move config files to under the `optuna_dashboard` directory.

### PR DESCRIPTION
**Pull request by c-bata**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

An alternative approach for #845.

## What does this implement/fix? Explain your changes.

Move config files to under the `optuna_dashboard` directory.

### CODE REVIEW

**Review by porink0424 — COMMENTED** [review]
Looks almost good, but we also have to exclude `typescript_tests/**` from `biome.js`?

**Comment by c-bata** [comment]
Thank you for your review! Good catch. I fixed the `biome.json`. Could you take a look again?

File: biome.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
 		"include": [
 			"optuna_dashboard/ts/**/*.ts",
 			"optuna_dashboard/ts/**/*.tsx",
-			"typescript_tests/**/*.ts",
-			"typescript_tests/**/*.tsx",
+			"optuna_dashboard/ts/tests/**/*.ts",
+			"optuna_dashboard/ts/tests/**/*.tsx",
```
Comment (porink0424) [inline biome.json:8]: Maybe we can just delete these (I think these are already included into two lines above these).

Comment (c-bata) [reply biome.json:8]: Ah, you are right 🙏

Comment (c-bata) [inline biome.json:8]:
```suggestion
```

**Pull request merged**